plugins/00_dokku-standard/install

```diff
@@ -33,7 +35,7 @@ After=docker.target
 [Service]
 Type=simple
 User=dokku
-ExecStart=/usr/local/bin/dokku ps:restore
+ExecStart=$dokku_path ps:restore
```

Comment:
binaries on arch usually reside in `/usr/bin/` - they had some huge restructurings some time ago, when they moved a lot to one location.